v_3_4_0/master_template.go

```diff
@@ -908,6 +908,20 @@ write_files:
       name: node-operator
       apiGroup: rbac.authorization.k8s.io
     ---
+    ## prometheus-external is prometheus from host cluster
+    kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: prometheus-external
```

Comment:
`-external` so that we dont collied for possible RBAC rules for prometheus that run in-cluster